packages/build/src/plugins/spawn.ts

```diff
@@ -37,15 +37,14 @@ const tStartPlugins = async function ({ pluginsOptions, buildDir, childEnv, logs
 export const startPlugins = measureDuration(tStartPlugins, 'start_plugins')
 
 const startPlugin = async function ({ pluginDir, nodePath, buildDir, childEnv }) {
-  const childProcess = execaNode(CHILD_MAIN_FILE, {
+  const childProcess = execaNode(CHILD_MAIN_FILE, [], {
```

Comment:
This are the types of `execaNode` seems like we where using it wrongly:

2nd argument here are the arguments passed down to the child process

![CleanShot 2022-11-28 at 10 30 19](https://user-images.githubusercontent.com/11156362/204242547-6c371e14-4de3-4091-866c-823079f31572.png)